content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -174,32 +181,28 @@ spec:
     name: busybox
 ```
 
-If there exists a headless Service in the same namespace as the Pod and with
-the same name as the subdomain, the cluster's DNS Server also returns an A or AAAA
-record for the Pod's fully qualified hostname.
-For example, given a Pod with the hostname set to "`busybox-1`" and the subdomain set to
-"`default-subdomain`", and a headless Service named "`default-subdomain`" in
-the same namespace, the Pod will see its own FQDN as
-"`busybox-1.default-subdomain.my-namespace.svc.cluster-domain.example`". DNS serves an
-A or AAAA record at that name, pointing to the Pod's IP. Both Pods "`busybox1`" and
-"`busybox2`" can have their distinct A or AAAA records.
+Given the above Service `"busybox-subdomain"` and the Pods which set `spec.subdomain`
+to `"busybox-subdomain"`, the first Pod will see its own FQDN as
+`"busybox-1.busybox-subdomain.my-namespace.svc.cluster-domain.example"`. DNS serves
+A and/or AAAA records at that name, pointing to the Pod's IP. Both Pods "`busybox1`" and
+"`busybox2`" will have their own address records.
 
 An {{<glossary_tooltip term_id="endpoint-slice" text="EndpointSlice">}} can specify
 the DNS hostname for any endpoint addresses, along with its IP.
 
 {{< note >}}
-Because A or AAAA records are not created for Pod names, `hostname` is required for the Pod's A or AAAA
+Because A and AAAA records are not created for Pod names, `hostname` is required for the Pod's A or AAAA
 record to be created. A Pod with no `hostname` but with `subdomain` will only create the
-A or AAAA record for the headless Service (`default-subdomain.my-namespace.svc.cluster-domain.example`),
-pointing to the Pod's IP address. Also, Pod needs to become ready in order to have a
+A or AAAA record for the headless Service (`busybox-subdomain.my-namespace.svc.cluster-domain.example`),
+pointing to the Pods' IP addresses. Also, the Pod needs to be ready in order to have a
 record unless `publishNotReadyAddresses=True` is set on the Service.
 {{< /note >}}
 
 ### Pod's setHostnameAsFQDN field {#pod-sethostnameasfqdn-field}
 
 {{< feature-state for_k8s_version="v1.22" state="stable" >}}
 
-When a Pod is configured to have fully qualified domain name (FQDN), its hostname is the short hostname. For example, if you have a Pod with the fully qualified domain name `busybox-1.default-subdomain.my-namespace.svc.cluster-domain.example`, then by default the `hostname` command inside that Pod returns `busybox-1` and  the `hostname --fqdn` command returns the FQDN.
+When a Pod is configured to have fully qualified domain name (FQDN), its hostname is the short hostname. For example, if you have a Pod with the fully qualified domain name `busybox-1.busybox-subdomain.my-namespace.svc.cluster-domain.example`, then by default the `hostname` command inside that Pod returns `busybox-1` and  the `hostname --fqdn` command returns the FQDN.
```

Comment:
Wrap this long line?

Reply:
ACK - whole doc could use a linewrap or ten